woocommerce/woocommerce-ios
# Purchase Details Mark I: Shipping & Download cells

This PR includes:
- the Shipping cell for simple and variable products
- the Downloads cell for simple and variable products

The next PR will include the purchase note cell.

## To test
- Choose a variety of products (simple, variable, and/or downloadable) from your store and create a new order. 
- Navigate to Orders > Order Number > Fulfill Order and select each type of product
- A grouped or affiliate product should not have any Purchase Details section
- a downloadable product will have the Downloads cell
- a normal product will have the Shipping cell

Update release notes:
